From: Takao Fujiwara <tfujiwar@redhat.com>
Date: Fri, 12 Feb 2021 20:45:26 +0900
Subject: [PATCH] xkbinfo: Update iso639Ids but not iso3166Ids correctly in
 evdev

Origin: https://gitlab.gnome.org/GNOME/gnome-desktop/-/commit/a56f1349
Bug-Debian: https://bugs.debian.org/987603
Bug-Ubuntu: https://launchpad.net/bugs/1925792

rxkb parser updates iso3166Ids by mistake for iso639Ids elements in evdev.

Resolves: https://gitlab.gnome.org/GNOME/gnome-desktop/-/issues/187
---
 libgnome-desktop/gnome-xkb-info.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgnome-desktop/gnome-xkb-info.c b/libgnome-desktop/gnome-xkb-info.c
index 61409210..b2eca699 100644
--- a/libgnome-desktop/gnome-xkb-info.c
+++ b/libgnome-desktop/gnome-xkb-info.c
@@ -258,7 +258,7 @@ add_layouts (GnomeXkbInfo        *self,
            iso639 = rxkb_iso639_code_next (iso639))
         {
           char *id = g_strdup (rxkb_iso639_code_get_code (iso639));
-          l->iso3166Ids = g_slist_prepend (l->iso3166Ids, id);
+          l->iso639Ids = g_slist_prepend (l->iso639Ids, id);
         }
       for (iso3166 = rxkb_layout_get_iso3166_first (layout);
            iso3166;
